src/components/Form/Login.tsx

```diff
@@ -48,8 +48,20 @@ const Login = () => {
     onSuccess: (data) => {
       const token = data.data?.accessToken;
       const expiry = data.data?.expiresIn;
-      setItem('h2t_access_token', token, expiry);
+      if (token && expiry) {
+        setItem('h2t_access_token', token, expiry);
       naviagte(redirect || '/');
+      }
+      else {
+        const type = data.data?.type;
+        const email = data.data?.email;
+        naviagte('/send-email',{
+          state: {
+            type: type,
+            email: email,
+          }
```

Comment:
Could you please explain this logic?

Reply:
When user haven't activated account -> server return email instead of access token and navigate to send activate email page